tgstation.dme

```diff
@@ -4890,13 +4890,16 @@
 #include "modular_skyrat\modules\ashwalkers\code\buildings\ash_farming.dm"
 #include "modular_skyrat\modules\ashwalkers\code\buildings\fuelwell.dm"
 #include "modular_skyrat\modules\ashwalkers\code\buildings\tendril_corruption.dm"
+#include "modular_skyrat\modules\ashwalkers\code\buildings\tendril_cursing.dm"
 #include "modular_skyrat\modules\ashwalkers\code\clothing\ash_armour.dm"
 #include "modular_skyrat\modules\ashwalkers\code\clothing\hands.dm"
 #include "modular_skyrat\modules\ashwalkers\code\clothing\head.dm"
 #include "modular_skyrat\modules\ashwalkers\code\clothing\misc.dm"
 #include "modular_skyrat\modules\ashwalkers\code\clothing\neck.dm"
 #include "modular_skyrat\modules\ashwalkers\code\clothing\shoes.dm"
 #include "modular_skyrat\modules\ashwalkers\code\clothing\under.dm"
+#include "modular_skyrat\modules\ashwalkers\code\effects\ash_ritual.dm"
+#include "modular_skyrat\modules\ashwalkers\code\effects\ash_rune.dm"
 #include "modular_skyrat\modules\ashwalkers\code\items\ash_centrifuge.dm"
 #include "modular_skyrat\modules\ashwalkers\code\items\ash_seedmesh.dm"
```

Comment:
because VSC is dumb and removed it (thanks autoticker), stick `#include "modular_skyrat\modules\ashwalkers\code\megafauna\_megafauna.dm"` between `#include "modular_skyrat\modules\ashwalkers\code\items\ashwalker_shaman.dm"` and `#include "modular_skyrat\modules\ashwalkers\code\species\Ashwalkers.dm"`

Reply:
that file does not exist.

Reply:
pull from master then, it exists

Reply:
ah yea, I assume its because something was merged prior to me starting.